neo/d3xp/GameEdit.cpp

```diff
@@ -1146,3 +1146,56 @@ void idGameEdit::MapEntityTranslate( const char *name, const idVec3 &v ) const {
 		}
 	}
 }
+
+
+/***********************************************************************
+
+  Debugger
+
+***********************************************************************/
+
+bool idGameEdit::IsLineCode( const char *filename, int linenumber ) const {
+	static idStr fileStr = idStr( MAX_PATH );
```

Comment:
why is this `static` and why `idStr( MAX_PATH )` ?

Reply:
this function does not allocate when that file string is made static, otherwise it does, and resizes also, thats the only reason i can think of now. 
bit silly optimize with an strcmp in there... removing it.